apps/base-docs/tutorials/docs/0_gassless-transactions-with-paymaster.md

```diff
@@ -0,0 +1,616 @@
+---
+title: 'Gasless Transactions on Base using Coinbase Paymaster'
+slug: /gasless-transaction-on-base-using-a-paymaster
+description: Learn how to leverage Coinbase Paymaster for seamless, gasless transactions on the Coinbase Cloud Developer Platform.
+author: hughescoin
+keywords:
+  [
+    Gas,
+    Gasless,
+    Transactions,
+    Paymaster,
+    Sponsor,
+    Sponsored Transactions,
+    Onchain,
+    Coinbase,
+    Base,
+    Crypto,
+    Cloud Platform,
+  ]
+tags: ['Gasless', 'Crypto', 'Gas', 'Sponsor', 'Coinbase', 'Base']
+difficulty: easy
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+# Gasless Transactions on Base using Coinbase Paymaster
+
+Still trying to onboard users to your app? Want to break free from the worries of gas transactions and sponsor them for your users on Base? Look no further!
+
```

Comment:
Coinbase Paymaster or Base Paymaster?

Good place for links as well.

Reply:
Good question ...

Reply:
Lets use Base!